docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,125 @@
+
+########################
+Debug cloud apps via SSH
+########################
+
+**Audience:** Users that want to debug their cloud apps on their own machine.
+
+SSH allows you to access all your Lightning components in real time.
+This can be useful if you execute your app in a different environment than you develop it.
+This is often the case with applications that rely on GPUs.
+
+You can inspect the entire file system and monitor the runtime environment.
+
+----
+
+************************
+Add SSH key to Lightning
+************************
+
+Before you can SSH to cloud machines, you will need to generate a new private SSH key, add it to the SSH agent, and add the public SSH key to your account on Lightning.
+The following steps are one-time setup to allow you to SSH into your cloud machines.
+
+
+Step 1: Create an SSH key
+==========================
+
+Open a terminal and run the following command (replace email with the address you used in your lightning.ai account):
+
+.. code:: bash
+
+   # make the ssh key (if you don't have one)
+   $ ssh-keygen -t ed25519 -C "your_email@example.com"
+
+This creates a new SSH key, using the provided email as a label.
+
+At the prompt, type a secure passphrase.
+
+.. code:: bash
+
+   > Enter passphrase (empty for no passphrase): [Type a passphrase]
+   > Enter same passphrase again: [Type passphrase again]
+
+
+Step 2: Add your key to Lightning
+=================================
+
+You can add SSH keys using Lightning.ai UI, or via this CLI command:
+
+.. code:: bash
+
+   $ lightning add ssh-key --public-key ~/.ssh/id_ed25519.pub
+
+Next, list your configured SSH keys:
```

Comment:
why this step?
if keeping it, explain how to validate

Reply:
this step is done to validate the SSH key was added properly. I'm fine dropping it.